presto-main/src/test/java/com/facebook/presto/sql/TestExpressionInterpreter.java

```diff
@@ -1075,6 +1076,14 @@ public void testLikeOptimization()
     {
         assertOptimizedEquals("unbound_string like 'abc'", "unbound_string = CAST('abc' AS VARCHAR)");
 
+        assertOptimizedEquals("unbound_string like '' escape '#'", "unbound_string like '' escape '#'");
```

Comment:
Uppercase SQL keywords `LIKE` and `ESCAPE` (the others instances here should have been uppercase, too)